TechnologyAssessmentApp/README.md

```diff
@@ -1,29 +0,0 @@
-# TechnologyAssessmentApp
-# MVC/Decorator/Facade designs pattern is used
-
-# Step to run the project
-## Go to working Directory
-## Open TCodingAssessmentApp.xcworkspace
-## use Command+B or Product+Build to build the project
-## Press Run icon in Xcode or Command+R to run the project on Simulator
-
-# Step to run the project from command-line
-## Make sure command-line tools of Xcode is installed 
-## Go to working Directory
-## xcodebuild -scheme TechnologyAssessmentApp build -allowProvisioningUpdates
-
-
-# Step to static code coverage 
-## Open TechnologyAssessmentApp.xcworkspace
-## use Command+Shift+B or Product+Analyze to analyze the project
-
-
-# Step to execute test cases
-## use Command+U or Product+Test to execute the test case
-
-# Fastlane is used for CI
-# Install fastlane through Terminal
-# brew cask install fastlane
-# export PATH="$HOME/.fastlane/bin:$PATH"
-
```

Comment:
Why are you deleting README file ? It's always good to add documentation about what is this repository about and how to use it.

Reply:
Readme File was redundant.So i deleted one.
Current Readme is at this place - https://github.com/kshitijgodara/SampleProject/blob/master/README.md

Reply:
ok my bad 👍